### ISSUE TITLE
Analysis page defaults to wrong tab

### ISSUE THREAD

**Issue by erikwaskiewicz** [AWGL/somatic_db#47]
Issue from Joanna - Sometimes the analysis page will default to the finalise page before any analysis has happened. ?first analysis works fine, it only happens after one analysis has been completed

Think the issue is from the #finalise at the end of the URL being carried from the 1st analysis back into the view samples page (where it does nothing), and then from there into the next sample analysis, where it redirects to the finalise tab

It's the javascript at the bottom of https://github.com/AWGL/somatic_db/blob/main/analysis/templates/analysis/analysis_sheet_dna.html that does the redirect, but there are cases where we want it to do that, a better fix would be to make sure that the view samples page removes any trailing hashes after redirection

**Comment by erikwaskiewicz** [comment]
Something like this looks like it'll do the job. Code to clear any trailing hashes from the URL, should be run on page load for the view_samples.html template
```
function remHash() {
         var uri = window.location.toString();
         if (uri.indexOf("#") > 0) {
            var clean_uri = uri.substring(0, uri.indexOf("#"));
            window.history.replaceState({}, document.title, clean_uri);
         }
      }
```
Taken from https://www.tutorialspoint.com/how-to-remove-the-hash-from-window-location-url-with-javascript-without-page-refresh

**Comment by erikwaskiewicz** [comment]
fixed in #57
